features/AMQP.feature

```diff
@@ -246,7 +246,7 @@ Feature: Test AMQP API
 
   @usingAMQP @cleanSecurity
   Scenario: login user
-    Given I create a user "user1" with id "user1-id"
+    Given I create a user "useradmin" with id "user1-id"
```

Comment:
"user1" with admin role renamed to "useradmin", to avoid conflicts with users created to test roles.